TODO.md

```diff
@@ -0,0 +1,17 @@
+
+
+# Temporary file, to log todos
+
+
+one pr for multiple image changes (?) need to handle this case later on
+add retry mechanism, if task fails after triggering the pr_script
+use GITHUB_TOKEN instead of GITHUB_USER and GITHUB_PASSWORD, for Hub
+test with real repo push (to test credentials)
+
+add configuration for all bh deploy image files
+
+
+jenkins:
```

Comment:
Planning to run on Jenkins instead of just a cron job?  Nice !  But just a cronjob is good enough for me but Jenkins would be a nice touch.

Reply:
Yes, I was planning to have this updater triggered via jenkins (with cronjob) at first - might change. 
Where would you host the updater with the cronjob, instead?

Reply:
Could even be on the same PAVICS machine itself.  It's just a cronjob, can be anywhere, as long as it not on some dev perso machine.

PAVICS have a mean to add more cronjob to its existing cronjob used for auto-deployment https://github.com/bird-house/birdhouse-deploy/blob/cf055ccf7c06439de36aba5686758ff37db2d864/birdhouse/env.local.example#L136-L140

So the image-updater cronjob only need to be deployed on 1 of the various PAVICS installations.  So only either Ouranos or CRIM will need to run this image-updater.  Unless we are running with different `config.json`, that's why I mentioned the path to the `config.json` should be configurable as well.